pkgs/development/python-modules/robotframework-seleniumlibrary/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, robotframework, selenium }:
+
+buildPythonPackage rec {
+  version = "3.2.0";
+  pname = "robotframework-seleniumlibrary";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "113ee01aa4fbfd322d4816e5704c0187ffcf6a047ea012a66d305f250ab485db";
+  };
+
+  # no tests included in PyPI tarball
+  doCheck = false;
```

Comment:
Perhaps try to get the tests working by using the github repo https://github.com/robotframework/SeleniumLibrary/tree/master/atest

Reply:
I see you had the same idea as I. :smiley: